ResourceStatusSystem/Command/DowntimeCommand.py

```diff
@@ -106,6 +106,13 @@ def _prepareCommand( self ):
       if not seHost:
         return S_ERROR( 'No seHost for %s' % elementName )
       elementName = seHost
+      
+    elif elementType == 'FTS':
+    	gocdbServiceType = "FTS"
+    	try:
+    		elementName	= getFTSServers("FTS3")[ 'Value' ][0]
+    	except:
+    		return S_ERROR( 'No FTS3 server specified in dirac.cfg (see Resources/FTSEndpoints)' )
```

Comment:
maybe there is another problem. Given that DTCommand queries GOCDB via GOCDBClient per each available resource/site, it needs a method to convert local DIRAC names (like 'FTS3-CERN') to GOCDB-compliant names (like 'https://fts3-lhcb.cern.ch:8443'). It seems getFTS3Servers is not the method I'm looking for, since it returns the list of FTS3 servers and not something like the dictionary of FTS3 server name keys with their URL values.